layouts/partials/carousel/host-mobile.css

```diff
@@ -41,7 +41,7 @@
     "right: 0;" +
     "bottom: 0;" +
     "left: 0;" +
-    "z-index: 1040;" +
+    "z-index: 110000;" +
```

Comment:
Why do we need to increase the z-index ?

Reply:
just was following recommendations from https://tvpage.atlassian.net/browse/DEV-8743

Reply:
@karloscastaneda : I don't think we need to increase the z-index.